packages/modules/electricity_tariffs/energycharts/tariff.py

```diff
@@ -12,7 +13,15 @@
 def fetch_prices(config: EnergyChartsTariffConfiguration) -> Dict[int, float]:
     current_dateTime = datetime.now()
     tomorrow = datetime.now() + timedelta(1)
-    start_time = current_dateTime.strftime("%Y-%m-%d") + 'T00%3A00%2B01%3A00'
+    if datetime.today().astimezone(pytz.timezone("Europe/Berlin")).dst().total_seconds()/3600:
+        # Sommerzeit if = 1
+        current_dateTime = current_dateTime - timedelta(hours=1)
+        start_time = current_dateTime.strftime("%Y-%m-%d") + 'T' + current_dateTime.strftime("%H") + \
+            '%3A00' + '%2B01%3A00'
+    else:
+        # keine Sommerzeit
+        start_time = current_dateTime.strftime("%Y-%m-%d") + 'T' + current_dateTime.strftime("%H") + \
+            '%3A00' + '%2B01%3A00'
```

Comment:
datetime.now() liefert die lokale Zeit, also unter Berücksichtigung von DST. Die Unterscheidung, ob Sommer-oder Winterzeit kann wegfallen.

> classmethod datetime.now(tz=None)
Return the current local date and time.

Reply:
Vom Server wird aber Winterzeit abgefragt,...daher ist es sonst immer eine Stunde verschoben. Die Daten vom Server sind Winterzeit.

> datetime.now() liefert die lokale Zeit, also unter Berücksichtigung von DST. Die Unterscheidung, ob Sommer-oder Winterzeit kann wegfallen.
> 
> > classmethod datetime.now(tz=None)
> > Return the current local date and time.

Die Daten vom Server sind Winterzeit. Deswegen ist es in der Sommerzeit immer um eine Stunde verschoben.

Reply:
Ich tippe eher auf UTC. Sollte aber in der API zu finden sein.

Reply:
> Ich tippe eher auf UTC. Sollte aber in der API zu finden sein.

Ja. DU hast Recht. Wurde angepasst (Sommerzeit +02:00 und Winterzeit +01:00).